.npmrc

```diff
@@ -0,0 +1,3 @@
+# Will prevent NPM from installing this package if the 
+# minimum engine versions are not met
+engine-strict=true
```

Comment:
I think we should include this so that installing the package in a project with an older node version fails more visibly.